.github/workflows/samples-jdk17.yaml

```diff
@@ -44,15 +48,15 @@ jobs:
           - samples/client/petstore/java-helidon-client/v3/se
           - samples/client/petstore/spring-http-interface-reactive
           - samples/client/petstore/spring-http-interface
+          - samples/client/petstore/spring-http-interface-reactive-noResponseEntity
+          - samples/client/petstore/spring-http-interface-noResponseEntity
           - samples/client/petstore/java/webclient-jakarta
           - samples/client/petstore/java/microprofile-rest-client-outer-enum
           # servers
           - samples/openapi3/server/petstore/springboot-3
           - samples/server/petstore/java-camel/
           - samples/server/petstore/java-helidon-server/v3/mp/
           - samples/server/petstore/java-helidon-server/v3/se
-          - samples/client/petstore/spring-http-interface-reactive
-          - samples/client/petstore/spring-http-interface
```

Comment:
What's the reason to remove these tests?

Reply:
they are duplicated. Same paths are already present in the #clients section. Probably copy/paste issue.